src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { PersonType } from '../types';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<PersonType[]>([]);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoadig] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoadig] = useState(true);
```
to avoid blinking when peoplePage loading